### ISSUE TITLE
programmatic signJar

### ISSUE THREAD

**Issue by bootstraponline** [wuntee/otertool#6]
```
    public static void signJar(String keystore, String keystorePassword, String jarFile, String alias) throws Exception{
        // TODO: make this programatic, vs calling jarsigner
```

Android uses [SignApk](https://github.com/android/platform_build/blob/master/tools/signapk/SignApk.java) which should be helpful. I modified it slightly for my [apks](https://github.com/bootstraponline/apks) project which just uses the default test cert.

**Comment by wuntee** [comment]
Yeah, I remember starting down that path at first. When I get some more free time ill look deeper into it. Thanks.
